<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestPaymentDialog</class>
 <widget class="QDialog" name="RequestPaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request Payment</string>
  </property>
  <widget class="QGroupBox" name="RequestPaymentBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>11</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Request Payment</string>
   </property>
   <widget class="QLabel" name="accountLabel">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>44</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Account:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accountComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>84</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceLabelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>86</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice Label:</string>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceAmountLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>129</y>
      <width>103</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice Amount:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceAmountLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>126</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newInvoiceButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Invoice</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="InvoiceDetailsBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>269</y>
     <width>661</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Invoice Details</string>
   </property>
   <widget class="QLabel" name="invoiceDetailsLabel_4">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>137</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceDetailsLabel_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>107</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Script:</string>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceDetailsLabel_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>75</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceDetailsLabelLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="invoiceDetailsLabel">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>41</y>
      <width>38</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Label:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceDetailsUrlLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>135</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceDetailsScriptLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>104</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="invoiceDetailsAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>72</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addressClipboardButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy To Clipboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scriptClipboardButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>103</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy To Clipboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="urlClipboardButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>133</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy To Clipboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="qrLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
